docs/01-Introduction.md

```diff
@@ -1,6 +1,82 @@
 # 1.0 - Introduction
 
-## 1.1 What is LORIS?
+### 1.1 What is LORIS-MRI?  LORIS-MRI is the backbone of the imaging
+component that makes up LORIS.
+ It is maintained in a separate repository so that it can be installed
+ on the file server and separated from the web server.
+These documents assume you have some knowledge regarding LORIS and
+a functioning installation. For information regarding LORIS itself,
+please consult the [LORIS wiki][1].
 
-## 1.2 How does LORIS work?
+LORIS-MRI is a set of libraries responsible for the insertion,
+visualization and archival of uploaded imaging datasets.
+ It expects an uploaded, compressed file containing
+several [DICOM][2] files, converts this into [MINC][3] and [NII][4]
+files, and then archives these files onto the server. Knowledge of
+these file formats can be helpful, but are not necessary for using or
+installing LORIS-MRI.
 
+LORIS-MRI allows you to easily organize and archive your imaging datasets
+and links them with
+ corresponding behavioral data in LORIS. Scans can be viewed and quality
+ controlled through the user's web browser, facilitating collaboration
+ between radiologists, clinicians and researchers.
+
+### 1.2 How does LORIS-MRI work?  ![user_story](images/user_story.png)
```

Comment:
Comments on the figure:
- I would remove the little guys. It takes a lot of space and does not add to the graph. Additionally, it is rare that the radiologist would do the QC of the images. They only check for incidental findings. In several studies, it would be someone else that does the QC of the images.
- in the attached screenshot are propositions for text edition
![user_story](https://user-images.githubusercontent.com/1402456/38314780-0c2ed31c-37f5-11e8-80ab-5c59b8f24360.png)